frontend/public/style/ancillary/_bootstrap-variables.scss

```diff
@@ -0,0 +1,874 @@
+$bootstrap-sass-asset-helper: false !default;
+//
+// Variables
```

Comment:
Wondering if we should prune any vars that are not being used?

Reply:
Unused vars removed